api/src/server/server.ts

```diff
@@ -225,12 +225,15 @@ export class Server extends ServerLoader {
   async $onReady() {
     this.expressApp.get("*", (req: Request, res: Response) => res.sendStatus(404));
 
-    if (process.env["S3_BUCKET_NAME"] === "ship-pacts") {
+    const params = await Params.getParams();
+
+    if (params.shipOutputBucket === "ship-pacts") {
       logger.info({msg: "Not creating bucket because the desired name is ship-pacts. Consider using a different bucket name to make this work."});
+    } else if (params.s3SkipEnsureBucket) {
```

Comment:
making this explicit. Maybe we can eventually remove the ship-pacts `if` branch altogether?